spec/models/manageiq/providers/openstack/infra_manager/refresher_rhos_juno_spec.rb

```diff
@@ -44,7 +44,7 @@
       EmsRefresh.refresh(@ems.network_manager)
       @ems.reload
 
-      @host = ManageIQ::Providers::Openstack::InfraManager::Host.all.detect { |x| x.name.include?('(NovaCompute)') }
+      @host = ManageIQ::Providers::Openstack::InfraManager::Host.all.order(:ems_ref).detect { |x| x.name.include?('(NovaCompute)') }
```

Comment:
Ok, I'll merge, but we should probably record the cassettes so this line is more consistent.  Maybe give the host we care about a very unique names and don't change it again.  That way, if we add another NovaCompute and re-record cassettes, we don't hit this again.